cyberduck/support.md

```diff
@@ -25,17 +25,16 @@ Please be aware that you are possibly using our software at no charge if you hav
 
 ## Logging Output
 
-```{note}
-Reveal the log output in `cyberduck.log` by choosing _Preferences → Connection → Log → _Show__.
-```
 
 ```````{tabs}
 ``````{group-tab} macOS
 
 `````{tabs}
 ````{tab} Log
 
-Log output can be found in the `cyberduck.log` file in `~/Library/Logs/Cyberduck`.
+Reveal the log output in `cyberduck.log` by choosing _Preferences → Connection → Log → _Show__.
```

Comment:
I would change "Reveal the log output in `cyberduck.log`" to "Reveal the log output `cyberduck.log`" or "Reveal the log output named `cyberduck.log`". The same applies to the text in the Windows section.

"Show" has extra underscores that aren't needed.